modules/candidate_parameters/jsx/monitoring.js

```diff
@@ -0,0 +1,449 @@
+/* exported RMonitoringStatus */
+
+var MonitoringStatus = React.createClass(
+  {
+    getInitialState: function() {
+      return {
+        flaggedOptions: {
+          yes: "Yes",
+          no: "No"
+        },
+        monitoredOptions: {
+          Y: "Yes",
+          N: "No"
+        },
+        Data: [],
+        formData: {},
+        updateResult: null,
+        errorMessage: null,
+        isLoaded: false,
+        loadedData: 0
+      };
+    },
+    componentDidMount: function() {
+      var that = this;
+      $.ajax(
```

Comment:
Can use `$.getJSON` to avoid boilerplate. Recommended by @jacobpenny 
Example here: https://github.com/aces/Loris/pull/2623/files#diff-fa0713b75cc8b35a892ae6ab9a7a9610R39